packages/jsii-diff/test/util.ts

```diff
@@ -12,10 +12,15 @@ export async function expectNoError(original: string, updated: string) {
 }
 
 export async function expectError(
-  error: RegExp,
+  error: RegExp | undefined,
   original: string,
   updated: string,
 ) {
+  if (error === undefined) {
```

Comment:
That's purely cosmetic, but I've been trying to be consistent with this particular notation:

```suggestion
  if (error == null) {
```